src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 rating">
+      <div class="stars"></div>
```

Comment:
Class names should be stars, stars--0, stars--1 in parent div ... and stars__star in child divs